index.html

```diff
@@ -1,19 +1,96 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
     <title>Document</title>
 </head>
+
 <body>
     <div class="flex-container">
-        <header class="header-items-bold">Header Placeholder</header>
-        <ul class="header-items">
+        <header class="header-items-bold">Header Logo</header>
+        <ul class="secondary-items">
```

Comment:
idk if the project says this but since the text is indicating it's a link, you can nest in an `a` tag in the child `li` to make it more link-like. For example, you'd get a mouse hover state that makes the links look clickable